rag/utils/es_conn.py

```
@@ -334,7 +334,7 @@ def search(
         self.logger.error(f"ESConnection.search timeout for {ATTEMPT_TIME} times!")
         raise Exception("ESConnection.search timeout.")
 
-    def insert(self, documents: list[dict], index_name: str, knowledgebase_id: str = None) -> list[str]:
+    def insert(self, documents: list[dict], index_name: str, knowledgebase_id: str = None, refresh: str | bool = "wait_for") -> list[str]:
```

Ruff (0.16.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)